<ui version="4.0" >
 <class>SentinelWrite_3Class</class>
 <widget class="QMainWindow" name="SentinelWrite_3Class" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SentinelWrite_3</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <widget class="QPushButton" name="pushButton" >
    <property name="geometry" >
     <rect>
      <x>440</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>1:FileOut</string>
    </property>
   </widget>
   <widget class="QLabel" name="tUsableyears" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>使用期間</string>
    </property>
   </widget>
   <widget class="QLabel" name="tCpyright" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>著作権認証番号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEND" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>END</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCopyright" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lUsableYears" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbFILEOPEN" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>FILE OPEN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDELETE" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>DELETE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbINSERT" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>INSERT</string>
    </property>
   </widget>
   <widget class="QLabel" name="tCerealNo" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>シリアルNo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lSerialNo" >
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget" >
    <property name="geometry" >
     <rect>
      <x>200</x>
      <y>10</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="frameShape" >
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="selectionMode" >
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior" >
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar" />
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
